Exception Handling/Leitura e Processamento de arquivo/src/Main.java

```diff
@@ -0,0 +1,27 @@
+public class Main {
+    
+    public static void main(String[] args){
+        File file = new File("Color.txt");
+        String pathAbs = file.getAbsolutePath();
+        
+        List<String> data = readFileReader(pathAbs);
```

Comment:
Jessika, não conseguimos encontrar o arquivo porquê dessa forma, estamos buscando no diretório raiz e lá não existe, teria que ser assim:
```suggestion
        File file = new File("");
        String path = file.getAbsolutePath().concat("/Exception Handling/Leitura e Processamento de arquivo/Color.txt");

        List<String> data = readFileReader(path);
```

ou mover o arquivo para a raiz :)

Reply:
Oiii @isdrchagas , muito obrigada pela observação!
Vou corrigir aqui e se puder verificar pra mim, por gentileza, super agradeço!

Reply:
aqui da forma que você fez ele ainda não consegue encontrar porquê no `pathAbs` não tem o caminho até o diretorio!!

Reply:
Ahh entendi! vou corrigir ;)